libraries/AP_RSSI/AP_RSSI.cpp

```diff
@@ -129,7 +130,7 @@ float AP_RSSI::read_receiver_rssi()
             receiver_rssi = read_channel_rssi();
             break;
         case RssiType::RSSI_RECEIVER : {
-            int16_t rssi = hal.rcin->get_rssi();
+            int16_t rssi = RC_Channels::get_rssi();
```

Comment:
I'm not sure about this one, I think that in this case we really want the RSSI provided by the receiver that is connected to the board - so we have to ask the HAL and not the library.

Reply:
maybe call it RC_Channels::get_receiver_rssi() ?

Reply:
Conclusion from dev call was to call it get_receiver_rssi (or something like that) to make clear what it does now.
We also discussed about merging AP_RSSI into RC_Channels and @tridge isn't against it, but we would need to be careful with moving the parameters. That's obviously something for the future.